src/shared/components/palette/document/write-document-field.component.ts

```diff
@@ -198,9 +220,17 @@ export class WriteDocumentFieldComponent extends AbstractFieldWriteComponent imp
 
   getUploadedFileName() {
     if (this.uploadedDocument) {
-      return this.uploadedDocument.get('document_filename').value;
+      return this.uploadedDocument.get(WriteDocumentFieldDefinitions.DOCUMENT_FILENAME).value;
     } else {
       return undefined;
     }
   }
 }
+
+class WriteDocumentFieldDefinitions {
```

Comment:
do we need a new class here? I think you could just declare those as `const` at the beginning of the file. Check for example: https://github.com/hmcts/ccd-case-ui-toolkit/blob/1241a6146fa1af8483545d11648510aef5d8e03e/src/shared/components/search-filters/search-filters.component.ts#L7

Reply:
DONE I created class members  static readonly UPLOAD_ERROR_FILE_REQUIRED